extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -44,18 +44,22 @@ export class NetCoreTool extends ShellExecutionHelper {
 
 	/**
 	 * This method presents the installation dialog for .NET Core, if not already present/supported
+	 * @param skipVersionSupportedCheck If true then skip the check to determine whether the .NET version is supported (for commands that work on all versions)
 	 * @returns True if .NET version was found and is supported
 	 * 			False if .NET version isn't present or present but not supported
 	 */
-	public async findOrInstallNetCore(): Promise<boolean> {
-		if ((!this.isNetCoreInstallationPresent || !await this.isNetCoreVersionSupported())) {
-			if (this.netCoreInstallState === netCoreInstallState.netCoreVersionSupported && vscode.workspace.getConfiguration(DBProjectConfigurationKey)[NetCoreDoNotAskAgainKey] !== true) {
-				void this.showInstallDialog();		// Removing await so that Build and extension load process doesn't wait on user input
-			} else if (this.netCoreInstallState === netCoreInstallState.netCoreVersionTooHigh && vscode.workspace.getConfiguration(DBProjectConfigurationKey)[NetCoreDowngradeDoNotShowAgainKey] !== true) {
-				void this.showDowngradeDialog();
+	public async findOrInstallNetCore(skipVersionSupportedCheck = false): Promise<boolean> {
+		if (!this.isNetCoreInstallationPresent || (this.isNetCoreInstallationPresent && !skipVersionSupportedCheck)) {
```

Comment:
Simplify to: `!(isPresent && skipCheck)`

Reply:
Merging this in so we can get a build out for bug bash, will follow up with simplifying this